app/models/mixins/ownership_mixin.rb

```diff
@@ -11,9 +11,27 @@ module OwnershipMixin
     virtual_column :owned_by_current_user,                :type => :boolean,    :uses => :evm_owner_userid
     virtual_column :owning_ldap_group,                    :type => :string,     :uses => :miq_group
     virtual_column :owned_by_current_ldap_group,          :type => :boolean,    :uses => :owning_ldap_group
+
+    # Determine whether to return vms ownded by the current user or not
+    # Requires `with_miq_group` scope to function
+    #
+    # Resulting SQL: "(LOWER(miq_groups.description) = 'some_miq_group'"
+    #
+    # Will result in the following when used with MiqExpression:
+    #
+    #   WHERE ((LOWER("miq_groups"."description") = 'some_miq_group') = 'true')
+    virtual_attribute :owned_by_current_ldap_group, :boolean, :arel => (lambda do |t|
+      ldap_group = User.current_user.try(:ldap_group).to_s.downcase
+
+      t.grouping(MiqGroup.arel_table[:description].lower.eq(ldap_group))
```

Comment:
could you use `arel_attribute`?
using that is why virtual attributes work at all

```ruby
t.grouping(MiqGroup.arel_attribute(:description).lower.eq(ldap_group))
```